<ui version="4.0" >
 <class>CompareBookingsClass</class>
 <widget class="QDialog" name="CompareBookingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NA05 Compare Bookings</string>
  </property>
  <widget class="QFrame" name="fraArticleNarcoticData" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1071</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QGroupBox" name="grpArticleData" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title" >
     <string>Article Data:</string>
    </property>
    <widget class="QLabel" name="lblArticleNo" >
     <property name="geometry" >
      <rect>
       <x>14</x>
       <y>22</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>ArticleNo.:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPackageUnit" >
     <property name="geometry" >
      <rect>
       <x>193</x>
       <y>22</y>
       <width>63</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>PackageUnit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblName" >
     <property name="geometry" >
      <rect>
       <x>387</x>
       <y>20</y>
       <width>31</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPharmaForm" >
     <property name="geometry" >
      <rect>
       <x>692</x>
       <y>22</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>PharmaForm:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDiffQtyInStock" >
     <property name="geometry" >
      <rect>
       <x>880</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Difference:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="articlecode" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>19</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="articlepackageunit" >
     <property name="geometry" >
      <rect>
       <x>263</x>
       <y>18</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="articlename" >
     <property name="geometry" >
      <rect>
       <x>423</x>
       <y>18</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="articlepharmaform" >
     <property name="geometry" >
      <rect>
       <x>790</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="diffqtyinstock" >
     <property name="geometry" >
      <rect>
       <x>950</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpNarcoticData" >
    <property name="geometry" >
     <rect>
      <x>1</x>
      <y>56</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title" >
     <string>Narcotic Data:</string>
    </property>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>12</x>
       <y>21</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>NarcoticGroup:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>21</y>
       <width>31</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13" >
     <property name="geometry" >
      <rect>
       <x>500</x>
       <y>21</y>
       <width>63</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>PackageUnit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12" >
     <property name="geometry" >
      <rect>
       <x>690</x>
       <y>21</y>
       <width>72</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text" >
      <string>Concentration:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="group" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="name" >
     <property name="geometry" >
      <rect>
       <x>240</x>
       <y>18</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="packageunit" >
     <property name="geometry" >
      <rect>
       <x>570</x>
       <y>18</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="concentration" >
     <property name="geometry" >
      <rect>
       <x>770</x>
       <y>18</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpSearchCriteria" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1001</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>Search:</string>
   </property>
   <widget class="QLabel" name="lblArticleNoSearch" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>19</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>ArticleNo.:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtArticleNoSearch" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>19</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboArticleNos" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtFromSearch" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>20</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="acceptDrops" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtToSearch" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>20</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFromSearch" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>23</y>
      <width>26</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>from:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblToSearch" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>23</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>to:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>18</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpNarcoticBookingsData" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>170</y>
     <width>491</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title" >
    <string>Narcotic Bookings:</string>
   </property>
   <widget class="QPushButton" name="btnRebuild" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string>A new dialog shows all deleted record sets which can be restored.</string>
    </property>
    <property name="text" >
     <string>Restore...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>430</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>In Iteration 1 nicht implementiert.</string>
    </property>
    <property name="text" >
     <string>Delete</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="narcotics" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount" >
     <number>11</number>
    </property>
    <property name="columnCount" >
     <number>6</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpOriginalicBookingsData" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>170</y>
     <width>501</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title" >
    <string>Pharmos Bookings:</string>
   </property>
   <widget class="BasarTableWidget" name="original" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount" >
     <number>11</number>
    </property>
    <property name="columnCount" >
     <number>6</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose" >
   <property name="geometry" >
    <rect>
     <x>930</x>
     <y>650</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
